src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs

```diff
@@ -619,9 +637,10 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                 }
 
                 return new SuggestedActionSet(
-                    refactoringSuggestedActions.ToImmutableAndFree(),
-                    SuggestedActionSetPriority.Low,
-                    applicableSpan);
+                    categoryName: PredefinedSuggestedActionCategoryNames.Refactoring,
+                    actions: refactoringSuggestedActions.ToImmutableAndFree(),
+                    priority: SuggestedActionSetPriority.Low,
+                    applicableToSpan: applicableSpan);
             }
 
             public async Task<bool> HasSuggestedActionsAsync(
```

Comment:
The API was created with the intention that GetSuggestedActionCategoriesAsync() would supersede HasSuggestedActionsAsync() entirely, so, it is safe to do so, but it is not explicitly promised anywhere in the API contract.

If it'll keep the code cleaner, it's probably ok.